src/main/java/org/reaktivity/nukleus/kafka/internal/stream/NetworkConnectionPool.java

```diff
@@ -320,6 +321,7 @@ public void flush(
         this.fetchPartitionMaxBytes = config.fetchPartitionMaxBytes();
         this.forceProactiveMessageCache = config.messageCacheProactive();
         this.readIdleTimeout = config.readIdleTimeout();
+        this.proactive = config.topicBootstrapEnabled();
```

Comment:
Should we rename `this.proactive` to `this.bootstrap` to avoid confusion with `messageCacheProactive` ?